CHANGES_NEXT_RELEASE

```diff
@@ -10,3 +10,4 @@
   * sinon-chai
   * grunt and grunt related module
   * closure-linter-wrapper
+- Add 'ruleName' as variable automatically in rule text field (EPL) on rule creation time (#208)
```

Comment:
Argg... issue number is #307, not #208.

My fault sorry :(

Reply:
Fixed in ef4dda6